app/features/notes_generator/metadata.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "notes_generator",
```

Comment:
The `metadata.json` attributes are still not aligned similarly as the other tools.